Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.cs

```diff
@@ -0,0 +1,165 @@
+        public Orientation Orientation
+        {
+            get { return (Orientation)GetValue(OrientationProperty); }
+            set { SetValue(OrientationProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="Orientation"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty OrientationProperty =
+            DependencyProperty.Register(
+                nameof(Orientation),
+                typeof(Orientation),
+                typeof(Menu),
+                new PropertyMetadata(Orientation.Horizontal, OrientationPropertyChanged));
+
+        /// <summary>
+        /// Identifies the <see cref="MenuFlyoutStyle"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty MenuFlyoutStyleProperty = DependencyProperty.Register(nameof(MenuFlyoutStyle), typeof(Style), typeof(MenuItem), new PropertyMetadata(default(Style)));
+
+        /// <summary>
+        /// Gets or sets the menu style for MenuItem
+        /// </summary>
+        public Style MenuFlyoutStyle
+        {
+            get { return (Style)GetValue(MenuFlyoutStyleProperty); }
+            set { SetValue(MenuFlyoutStyleProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="TooltipStyle"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty TooltipStyleProperty = DependencyProperty.Register(nameof(TooltipStyle), typeof(Style), typeof(Menu), new PropertyMetadata(default(Style)));
+
+        /// <summary>
+        /// Gets or sets the tooltip styles for menu
+        /// </summary>
+        public Style TooltipStyle
+        {
+            get { return (Style)GetValue(TooltipStyleProperty); }
+            set { SetValue(TooltipStyleProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="TooltipPlacement"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty TooltipPlacementProperty = DependencyProperty.Register(nameof(TooltipPlacement), typeof(PlacementMode), typeof(Menu), new PropertyMetadata(default(PlacementMode)));
+
+        /// <summary>
+        /// Gets or sets the tooltip placement on menu
+        /// </summary>
+        public PlacementMode TooltipPlacement
+        {
+            get { return (PlacementMode)GetValue(TooltipPlacementProperty); }
+            set { SetValue(TooltipPlacementProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="AllowTooltip"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty AllowTooltipProperty = DependencyProperty.Register(nameof(AllowTooltip), typeof(bool), typeof(Menu), new PropertyMetadata(false));
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to allow tooltip on alt or not
+        /// </summary>
+        public bool AllowTooltip
```

Comment:
Maybe we can exclude AccessKey from AllowTooltip but this property is required for anyone using InputGesture especially if they have an OS version less than AU, how can they decide to show or hide tooltips (AccessKeyManager is introduced in AU) ?